packages/wrangler/src/event-subscriptions/products.ts

```diff
@@ -0,0 +1,74 @@
+import type { HandlerContext } from "../core/types";
+
+// Use the same `id` here as the one you used in queue-broker-worker
+export const PRODUCTS: ProductSpec[] = [
+	{
+		id: "clipboards",
+		name: "Clipboards",
+		resource: { name: "clipboard" },
+		subscription: {
+			create: (
+				ctx,
+				{ accountId, name, resourceId: clipboardId, queueId, events }
+			) =>
+				ctx.fetchResult<{ id: string }>(
+					`/accounts/${accountId}/clipboard/subscriptions`,
+					{
+						method: "POST",
+						body: JSON.stringify({
+							name,
+							clipboardId,
+							queueId,
+							events,
+						}),
+					}
+				),
+		},
+	},
+	{
+		id: "workersAI",
+		name: "Workers AI",
+		resource: null,
+		subscription: {
+			create: () => Promise.resolve({ id: "" }),
```

Comment:
These will be populated by teams. I'll remove entries for teams who do not onboard right now.